File: commonLib/src/main/java/com/travel/commonlib/hotel/PassengerModel.java
package com.travel.commonlib.hotel;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;
import com.travel.jumbo.model.hotel.PassengerDetail;

import java.io.Serializable;
import java.util.List;

public class PassengerModel implements Serializable {
    @SerializedName("PassengerDetails")
    @Expose
    private List<PassengerDetail> passengerDetails = null;
    @SerializedName("CountryCodes")
    @Expose
    private List<CountryCode> countryCodes = null;

    public List<PassengerDetail> getPassengerDetails() {
        return passengerDetails;
    }

    public void setPassengerDetails(List<PassengerDetail> passengerDetails) {
        this.passengerDetails = passengerDetails;
    }

    public List<CountryCode> getCountryCodes() {
        return countryCodes;
    }

    public void setCountryCodes(List<CountryCode> countryCodes) {
        this.countryCodes = countryCodes;
    }

}

File: commonLib/src/main/java/com/travel/commonlib/flight/PaxModel/PassengerModel.kt
package com.travel.commonlib.flight.PaxModel

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import java.io.Serializable
import java.util.*

class PassengerModel : Serializable {
    @SerializedName("IsLoggedIn")
    @Expose
    var loggedIn: Boolean? = null

    @SerializedName("UserName")
    @Expose
    var userName: String? = null

    @SerializedName("CountryList")
    @Expose
    var countryList: Any? = null

    @SerializedName("PassengerInfos")
    @Expose
    var passengerInfos: ArrayList<PassengerInfo>? = null

    @SerializedName("SSRDetails")
    @Expose
    private var sSRDetails: String? = null

}